lib/firespring_dev_commands/aws/route53.rb

```diff
@@ -0,0 +1,72 @@
+require 'aws-sdk-route53'
+
+module Dev
+  class Aws
+    # Class for performing Route53 functions
+    class Route53
+      attr_reader :client, :zones
+
+      def initialize
+        @client = ::Aws::Route53::Client.new
+        @zones = []
+      end
+
+      def get_hosted_zones(domains)
+        if domains.empty?
+          response = client.list_hosted_zones
+          response.hosted_zones.each do |hosted_zone|
+            @zones << hosted_zone.id
+          end
+        else
+          domains.each do |domain_name|
+            zone = client.list_hosted_zones_by_name({dns_name: domain_name, max_items: 1})
+            target_name = zone.hosted_zones[0].name.chomp!('.') if zone.hosted_zones[0].name.end_with?('.')
+            @zones << zone.hosted_zones[0].id unless target_name != domain_name
```

Comment:
Should this raise an error? Not sure if we would want the script to continue here or not?